### PR TITLE
Feature/#56/store images

### LINKED ISSUE #56: Store images for rooms when new room is created
## User story

**Who:** **As a** user

**What:** **I want to** be able to upload an image to a room

**Why:** **so that we can** all rooms have images

## Implementation details
- When creating a new room there should be a possibility to save an image
-  This story is relevant: https://github.com/HackYourFuture-CPH/chattie/issues/55 and this one: https://github.com/HackYourFuture-CPH/chattie/issues/53
- When this task is done it should be possible to add an image to a room
- Use the function created in [this story](https://github.com/HackYourFuture-CPH/chattie/issues/66) to save the image and get the url

### PR DESCRIPTION
**Pull request by LucyChyzhova**
# Description

Created a new Component "AddImageToRoom"
Added storybook to this component
Started to style it.

Fixes #56  

# How to test?
Tests by storybook
Click on Add Image to room => component

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/AddImageToRoom/AddImageToRoom.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <div className="add-image-to-room">
+      <input
+        type="file"
+        accept="image/*"
+        onChange={handleImageUpload}
+        ref={imageUploader}
+      />
+
+      <div className="image-container ">
+        <div className="button">
```
Comment (tpetrina) [inline src/client/components/AddImageToRoom/AddImageToRoom.js:31]: This should be `button-container` to be more specific.
Or be more concrete and use `upload-image-button`.

File: src/client/components/AddImageToRoom/AddImageToRoom.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          <button type="button" onClick={() => imageUploader.current.click()}>
+            +
+          </button>
+        </div>
+
+        <div className="input-picture">
```
Comment (tpetrina) [inline src/client/components/AddImageToRoom/AddImageToRoom.js:37]: Suggestion: name it `upload-preview` or `image-preview`.

File: src/client/components/AddImageToRoom/AddImageToRoom.styles.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.add-image-to-room input {
+  display: none;
+}
+
+.image-container {
```
Comment (tpetrina) [inline src/client/components/AddImageToRoom/AddImageToRoom.styles.css:5]: I suggest all selectors start with `.add-image-to-room ` to properly scope this.

File: src/client/components/AddImageToRoom/AddImageToRoom.stories.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import AddImageToRoom from './AddImageToRoom';
+
+export default { title: 'Add Image To Room' };
+
+export const Component = () => <AddImageToRoom AddRoomImage={() => {}} />;
```
Comment (tpetrina) [inline src/client/components/AddImageToRoom/AddImageToRoom.stories.js:6]: Are you planning to add prop `AddRoomImage` in the future? If not, this is extra.

**Review by benna100 — APPROVED** [review]

**Pull request merged**